FinalXmL/FinalXmL/Pages/Index.cshtml.cs

```diff
@@ -9,34 +9,38 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using Newtonsoft.Json.Schema;
-using QuickType;
-using QuickType1;
+using QuickTypeCountry;
+using QuickTypeGeo;
 
 namespace FinalXmL.Pages
 {
     public class IndexModel : PageModel
     {
         public void OnGet()
         {
-            using (var webClient = new WebClient())
+            //hard coding values used in a function is a bad practice especially when you have to change every instance in functions rather than changing one variable name
+            //also if we wanted to manipulate and edit parts of the endURL creating this variable would make that process easier
```

Comment:
Ditto comment above... save the discussion/justification/commentary for the code review.